routes/sunglare/sunglare_predictiveReport.js

```diff
@@ -0,0 +1,171 @@
+// sunglare_predictiveReport: Generates a report for the sunglare module
+
+const burgerHelper = require('../../helper_functions/sunglare_helperFunctions');
```

Comment:
this is not a descriptive name :(
also the nested functions inside here CreateLocationClause can be updated to constructors naming convention